source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -2,6 +2,42 @@
 
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
+## Release 2021-01-12
+
+### Highlights
+ - Pre-packaged and pre-installed Mattermost Incident Management v1.2.0.
+
+### Improvements
+
+#### User Interface (UI)
+ - Changed the number of file attachments allowed per post, from 5 to 10.
+ - Enabled the **More options** menu on channels in the new sidebar to move multi-selected groups of channels.
```

Comment:
@andrewbrown00 Would something like this work better:

"Added support for multi-selecting channels via the **More options** menu in the channel sidebar to be able to move multi-selected groups of channels to a category."

Reply:
@amyblais - Was multi-select channel support in LHS added to a previous Cloud release changelog? I'm documenting this functionality for release v5.31, which suggests to me that it's already released for Cloud.

@amyblais - I'd avoid a statement that implies that selecting multiple channels at a time is something that happens through the **More options** menu. Instead, I'd recommend something along the lines of:

"Added support to move multi-selected groups of channels to another category via the **More options** menu.

Thoughts?

Reply:
Yes, I see, multi-selection in the changelog for the previous versioning: https://docs.mattermost.com/cloud/cloud-administration/cloud-changelog.html.

In that case your suggestion for wording makes sense, thanks!

Reply:
Thanks for the quick feedback and suggestions here 🙂 

@amyblais let's go with @cwarnermm's suggestion:

> Added support to move multi-selected groups of channels to another category via the More options menu.